### DISCUSSION TITLE
Tolerance value in simplify function

### DISCUSSION THREAD

**Discussion by raksha-agarwal** [shapely/shapely#1478] [Q&A]
Kindly specify details about tolerance value  (or expose the code) for simplify function. What is the unit for the tolerance level? 

I am using this for simplifying GPS coordinates.

**Comment by mwtoews** [comment]
From the [user manual for `simplify`](https://shapely.readthedocs.io/en/stable/manual.html#object.simplify):
> All points in the simplified object will be within the _tolerance_ distance of the original geometry.

So if your GPS coordinates are projected in a CRS with length units of metres, then the tolerance is also meters. If they are lat/long, then the units are in degrees, and should ideally be reprojected to (e.g.) a UTM zone. There's a useful animation [here](https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm#Algorithm) that shows how the tolerance distance is used with the Douglas–Peucker algorithm.

**Comment by raksha-agarwal** [comment]
Thank you for the reply.
I am using it as shown here.
https://geoffboeing.com/2014/08/reducing-spatial-data-set-size-with-douglas-peucker/

My data is reduced by 30% if tolerance is set to 1e-05. 
I am a bit unsure about the unit of this tolerance.
